deploy/charts/cert-manager/templates/psp-clusterrole.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.global.rbac.pspEnabled }}
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ template "cert-manager.fullname" . }}-psp
+  labels:
+    app: {{ include "cert-manager.name" . }}
+    chart: {{ include "cert-manager.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+rules:
+- apiGroups: ['extensions']
+  resources: ['podsecuritypolicies']
```

Comment:
If we change this to `policy/v1beta1`, make sure to update it here too 😄

Reply:
Since 1.10 so we should be good.